src/utils/timeSeriesResultsByOption.js

```diff
@@ -68,6 +69,9 @@ export function getMonthly ({
         shorterOption.mortgageRate,
         investmentRate,
         inflationRate,
+        month >= shorterOption.mortgageTerm * COMPOUND_FREQUENCY
+          ? shorterOptionPMT
+          : 0,
         monthlyResultsByOption.shorter[month - 1]
```

Comment:
for thee `shorterOption` the `investmentPMT` is equal to the `shorterOptionPMT` only if  the month is higher or equal to the term